changelogs/unreleased/gh-8605-add-gethostname.md

```diff
@@ -0,0 +1,3 @@
+## feature/box
```

Comment:
I'm ready to push, but there is a conflict. Please, rebase on a fresh master and set the full-ci label.

Reply:
Done.

Reply:
Waiting for the test fix.

Reply:
@Totktonada, the test is fixed, PR ready for merge.